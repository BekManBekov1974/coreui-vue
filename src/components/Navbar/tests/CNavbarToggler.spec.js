import { mount } from '@vue/test-utils'
import Component from '../CNavbarToggler'

const ComponentName = 'CNavbarToggler'
const defaultWrapper = mount(Component)
const customWrapper = mount(Component, {
  props: {
    tag: 'div'
  },
  attrs: {
    type: 'submit'
  },
  slots: {
    default: 'CNavbar items'
  }
})

describe(ComponentName, () => {
  it('has a name', () => {
    expect(Component.name).toBe(ComponentName)
  })
  it('renders correctly', () => {
    expect(defaultWrapper.element).toMatchSnapshot()
  })
  it('renders correctly', () => {
    expect(customWrapper.element).toMatchSnapshot()
  })
})
