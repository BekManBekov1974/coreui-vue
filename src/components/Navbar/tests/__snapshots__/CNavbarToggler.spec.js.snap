// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CNavbarToggler renders correctly 1`] = `
<button
  class="navbar-toggler"
  type="button"
>
  <span
    class="navbar-toggler-icon"
  />
</button>
`;

exports[`CNavbarToggler renders correctly 2`] = `
<button
  class="navbar-toggler"
  type="submit"
>
  CNavbar items
</button>
`;
